pkg/elasticsearch/indices.go

```diff
@@ -95,9 +96,23 @@ func (ec *esClient) AddAliasForOldIndices() bool {
 		}
 
 		if payload.ResponseBody[index] != nil {
-			indexBody := payload.ResponseBody[index].(map[string]interface{})
+			indexBody, ok := payload.ResponseBody[index].(map[string]interface{})
+			if !ok {
+				logger.Warnf("unable to unmarshal index '%s' response body for cluster '%s'. Type: %s",
+					index,
+					ec.cluster,
+					reflect.TypeOf(payload.ResponseBody[index]).String())
```

Comment:
You can print the type more succinctly (same result) with the %T format directive like this:
```
logger.Warnf("unable to unmarshal index '%s' response body for cluster '%s'. Type: %T", index, ec.cluster,
ec.cluster)
```